src/services/calendar.ts

```diff
@@ -125,15 +125,16 @@ export class iCloudCalendarService {
     dsid: string;
     dateFormat = "YYYY-MM-DD";
     calendarServiceUri: string;
+    tz = dayjs.tz.guess() || "US/Pacific";
```

Comment:
should we change the default to `UTC` instead? i think its more common to have that as the default

Reply:
Yep, but i think "UTC" we should leave as a fallback.